Musicplayer.css

```diff
@@ -8,62 +8,32 @@ body{
 
 .grid-container{
 	display: grid;
-	grid-template-columns: 15% 10% 75%;
-	grid-template-rows: auto auto auto auto auto;
+	grid-template-columns: 15% 35% 50%;
+	grid-template-rows: 58px auto auto auto auto;
 	height: 100vh;
 }
 
-.img_1{
-	grid-column-start: 1;
-	grid-column-end: span 4;
-	height:300px;
-	background:red;
-}
-
-.img_1 img{
-	display:block;
-	margin-left:auto;
-	margin-right:auto;
-	height:300px;
-	width:100%;
-}
-
-.footer{
-	background-color: white;
-	grid-column-end: span 4;
-	padding-top:25px;
-	text-align: center;
-	color:black;
-	height:50px;
-}
-
 .navigation{
 	box-sizing: border-box;
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	margin-top:-10px;
```

Comment:
21/9: Unlike layout one, in layout two I decided to make the apply the navbar at the top so I decided to remove of of the setting in the navigation div box. I will setting the color latter in the nav.